modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeGetImagePreviewURLMethodCheck.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeGetImagePreviewURLMethodCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if (!fileName.endsWith(".java") && !fileName.endsWith(".jsp")) {
+			return content;
+		}
+
+		boolean replaced = false;
+
+		Matcher getImagePreviewURLMatcher = _getImagePreviewURLPattern.matcher(
+			content);
+
+		while (getImagePreviewURLMatcher.find()) {
+			String methodCall = getImagePreviewURLMatcher.group();
+
+			content = StringUtil.replace(
+				content, methodCall,
+				StringUtil.replace(methodCall, "DLUtil", "_dlURLHelper"));
+
+			replaced = true;
+		}
+
+		if (fileName.endsWith(".java") && replaced) {
+			content = JavaSourceUtil.addImports(
+				content, "com.liferay.document.library.util.DLURLHelper");
```

Comment:
Rather than add the import, you may want replace the import `com.liferay.document.library.kernel.util.DLUtil` instead, unless you are going to add additional methods. If not, then replace the import. Checking to make sure the import exists in the `java` case may also be a good idea.

Reply:
Hello @kevhlee!

My concern with this is if this import is to be used in other calls the method.
What do you think?

Reply:
Fair enough.